src/openapiGenerator/options/customOptions.ts

```diff
@@ -76,6 +76,10 @@ const templates: Array<SEBTemplate> = [
     {
         generator: "typescript-axios",
         templatePath: "./node_modules/@sebgroup/frontend-tools/dist/openapiGenerator/templates/typescript-axios"
+    },
+    {
+        generator: "typescript-angular",
+        templatePath: "./node_modules/@sebgroup/frontend-tools/dist/openapiGenerator/templates/typescript-angular"
```

Comment:
So `typescript-angular` here uses `HttpClientModule` ? Just wondering.

Reply:
yep 

see this example of a generated file https://github.com/OpenAPITools/openapi-generator/blob/ec39005ae528f88522caedd91db91c0b59624d94/samples/client/petstore/typescript-angular-v6-not-provided-in-root/builds/default/api/pet.service.ts#L36